extensions/elytron-security-jdbc/runtime/src/main/java/io/quarkus/elytron/security/jdbc/JdbcSecurityRealmConfig.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.elytron.security.jdbc;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * A configuration object for a jdbc based realm configuration,
+ * {@linkplain org.wildfly.security.auth.realm.jdbc.JdbcSecurityRealm}
+ */
+@ConfigRoot(name = "security.jdbc", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class JdbcSecurityRealmConfig {
+
+    /**
+     * The authentication mechanism
+     */
+    @ConfigItem(defaultValue = "BASIC")
+    public String authMechanism;
+
+    /**
+     * The authentication mechanism
+     */
+    @ConfigItem(defaultValue = "Quarkus")
+    public String realmName;
+
+    /**
+     * If the properties store is enabled.
+     */
+    @ConfigItem
+    public boolean enabled;
```

Comment:
Should it be set to true by default ? Otherwise the users who need it have to 1) add the module 2) enable the feature while the users who don't need the feature don't have to do anything :-)